app/views/candidate_interface/sign_in/expired.html.erb

```diff
@@ -2,15 +2,15 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= form_with model: @candidate, url: candidate_interface_sign_in_path, method: :post do |f| %>
+    <%= form_with model: @candidate, url: candidate_interface_create_expired_sign_in_path, method: :post do |f| %>
```

Comment:
Does this need to be a form with `@candidate`? Seems like we don't need the `@candidate` at all here?